stock_quant_status_update/wizards/stock_quant_status_update_wizard.py

```diff
@@ -21,6 +21,6 @@ def action_stock_quant_status_update(self):
             if self.yahoo_product_state_id.id == int(
                 self.env["ir.config_parameter"]
                 .sudo()
-                .get_param("stock_ext_sst.picking_product_state_id")
+                .get_param("stock_quant_status_update.picking_product_state_id")
             ):
```

Comment:
This is bad. Move this part outside the for loop.

Add a migration script to update the param key, or communicate with @kanda999 to cover this part manually.